platform/cxxsupport/mstd_cstddef

```diff
@@ -42,6 +47,14 @@
 #define MSTD_CONSTEXPR_OBJ_14 const
 #endif
 
+#if __cplusplus >= 201103
```

Comment:
Don't actually need this `#if` here, as we are already assuming C++11 if C++ now.